pkgs/applications/virtualization/docker/default.nix

```diff
@@ -11,14 +11,14 @@ with lib;
 
 stdenv.mkDerivation rec {
   name = "docker-${version}";
-  version = "1.13.1";
-  rev = "092cba3"; # should match the version commit
+  version = "17.03.0-ce";
+  rev = "60ccb22"; # should match the version commit
```

Comment:
I'd a comment to that effect then, since the variable isn't used anywhere in Nix and looks like just a leftover of a previous factoring of the expression (I thought someone was using `inherit rev` in the `fetchFromGitHub` call below and then stopped doing that.